doc/errors.rst

```diff
@@ -47,13 +47,30 @@ One case that is not easily covered by the assert statement is asserting Excepti
 
 	  assert caught.exception.param == 'some_value'
 
+:func:`slash.assert_raises` will raise ``TestFailed`` exception in case the expected exception was not raised:
+
+.. code:: python
+
+	 >>> with slash.assert_raises(Exception) as caught: # doctest: +IGNORE_EXCEPTION_DETAIL
+	 ...    pass
+	 Traceback (most recent call last):
+	     ...
+	 TestFailed: ...
+
+In case the test should allow specific exception, but not enfore it (ex. timing issue), you can use :func:`slash.allow_raises`:
```

Comment:
About the note & link, There is this note:
https://github.com/getslash/slash/commit/68f90879d67892194d4452d27861c831f9fa91d7#diff-c430fdc89298c75a04be79e1cc7e2283R73
What should I add except for this?